03_bag.ipynb

```diff
@@ -550,75 +521,95 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pandas is faster than Pure Python.  In the same way, `dask.dataframe` can be faster than `dask.bag`.  We will work more with dataframes later, but from for the bag point of view, they are frequently the end-point of the \"messy\" part of data ingestion - once the data can be made into a data-frame, then complex split-spply-combine logic will become much more straight-forward and efficient.\n",
+    "For the same reasons that Pandas is often faster than pure Python, `dask.dataframe` can be faster than `dask.bag`.  We will work more with DataFrames later, but from for the bag point of view, they are frequently the end-point of the \"messy\" part of data ingestion—once the data can be made into a data-frame, then complex split-apply-combine logic will become much more straight-forward and efficient.\n",
```

Comment:
I mean, then should in hyphenated words, of course 😜

But even for ranges, e.g "The flight takes 8–10 hours" the n-dash is more appropriate... happily LA to Seattle is only 2 and some hours though.